src/sentry/static/sentry/app/components/errorRobot.tsx

```diff
@@ -89,13 +92,12 @@ class ErrorRobot extends React.Component {
             project={project}
             source="issues_list"
             disabled={!project}
-            title={!project ? t('Select a project to create a sample event') : null}
```

Comment:
Does `CreateSampleEventButton` explicitly check for `null`?

Reply:
It does not